packages/build-info/src/frameworks/nuxt.ts

```diff
@@ -34,11 +34,6 @@ export class Nuxt extends BaseFramework implements Framework {
         this.build.command = `nuxt build`
         this.build.directory = `dist`
         this.dev.command = `nuxt dev`
-
-        this.env = {
-          AWS_LAMBDA_JS_RUNTIME: 'nodejs18.x',
-          NODE_VERSION: '18',
```

Comment:
Nuxt has no strict requirement for node 18 and prefers LTS (or 22)
https://github.com/nuxt/nuxt/blob/main/packages/nuxt/package.json#L155-L157